lib/NNlibCUDA/test/activations.jl

```diff
@@ -0,0 +1,5 @@
+@testset "Broadcast Fix" begin
+    if CUDA.has_cudnn()
+        @test testf(x -> logσ.(x), rand(5))
```

Comment:
I don't know what this `ftest` is doing, would be good to have some internal testing utils instead